project/test/resources/OPConsultantRecord-Output.json

```diff
@@ -1,2688 +1,2143 @@
+          "code": "vital-signs",
+          "display": "Vital Signs"
+        } ],
+        "text": "Vital Signs"
+      } ],
+      "code": {
+        "coding": [ {
+          "system": "http://loinc.org",
+          "code": "29463-7",
+          "display": "weight"
+        } ],
+        "text": "Weight"
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "effectiveDateTime": "2024-07-01T16:29:32+05:30",
+      "valueQuantity": {
+        "value": 62.0,
+        "unit": "kg",
+        "system": "http://unitsofmeasure.org",
+        "code": "kg"
+      }
+    }
+  }, {
+    "fullUrl": "urn:uuid:64482fc0-c5c0-4f49-8dc7-8632ea71c6db",
+    "resource": {
+      "resourceType": "Observation",
+      "id": "64482fc0-c5c0-4f49-8dc7-8632ea71c6db",
+      "meta": {
+        "versionId": "0",
+        "lastUpdated": "2024-07-01T16:29:32.573+05:30",
+        "profile": [ "https://nrces.in/ndhm/fhir/r4/StructureDefinition/Observation" ]
+      },
+      "status": "final",
+      "category": [ {
+        "coding": [ {
+          "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code": "vital-signs",
+          "display": "Vital Signs"
+        } ],
+        "text": "Vital Signs"
+      } ],
+      "code": {
+        "coding": [ {
+          "system": "http://loinc.org",
+          "code": "39156-5",
+          "display": "Body mass index (BMI) [Ratio]"
+        } ],
+        "text": "Body Mass Index"
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "effectiveDateTime": "2024-07-01T16:29:32+05:30",
+      "valueQuantity": {
+        "value": 62.0,
+        "unit": "kg/m2",
+        "system": "http://unitsofmeasure.org",
+        "code": "kg/m2"
+      }
+    }
+  }, {
+    "fullUrl": "urn:uuid:9d7e0de1-e040-42ea-a246-76aa6d85e34b",
+    "resource": {
+      "resourceType": "Observation",
+      "id": "9d7e0de1-e040-42ea-a246-76aa6d85e34b",
+      "meta": {
+        "versionId": "0",
+        "lastUpdated": "2024-07-01T16:29:32.573+05:30",
+        "profile": [ "https://nrces.in/ndhm/fhir/r4/StructureDefinition/Observation" ]
+      },
+      "status": "final",
+      "category": [ {
+        "coding": [ {
+          "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code": "vital-signs",
+          "display": "Vital Signs"
+        } ],
+        "text": "Vital Signs"
+      } ],
+      "code": {
+        "coding": [ {
+          "system": "http://loinc.org",
+          "code": "882-1",
+          "display": "ABO and Rh group [Type] in Blood"
+        } ],
+        "text": "Blood Group"
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "valueCodeableConcept": {
```

Comment:
ABDM observations support both SNOMED and LOINC coding. However, I wouldn't suggest mixing SNOMED/LOINC as code and value. In this particular case, if you want to use LOINC as observation.code, then use valueCodeableConcept also from LOINC. You may check the answers from https://loinc.org/882-1/

alternatively, if you should use SNOMED- use observation.code = 112143006 (ABO group phenotype)
SCTID: